packages/forma-36-website/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "@typescript-eslint/no-explicit-any": "off"
```

Comment:
When I add components from alpha I get this errors again. I didn't know how to manage it in any other way then this one.

Reply:
Even after the change @wichniowski made???

Reply:
sorryyyyy, something was not up to date on my side. reomved.

Reply:
yeah, this is me again 😀